### PR TITLE
add a remote cluster check step to "verify the installation" multicluster setup docs

### LINKED ISSUE #15911: call out istioctl create-remote-secret interaction with rancher proxy
Rancher has a [feature](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/manage-clusters/access-clusters/authorized-cluster-endpoint#two-authentication-methods-for-rke-clusters) which proxies kubectl traffic through a different URL. When using this feature the kubeconfig created by `istioctl create-remote-secret` gets created with a `certificate-authority-data` CA bundle which (can be) completely unrelated to the cert used to terminate TLS at the rancher proxy endpoint. In this scenario istiod fails to sync and produces x509 unknown authority errors.

We should call this out in the docs somewhere. Cutting this issue mostly as a reminder to myself.

### PR DESCRIPTION
**Pull request by rsalmond**
## Description

Add a step to the [verify the installation](https://istio.io/latest/docs/setup/install/multicluster/verify/) multicluster setup docs for executing `istioctl remote clusters`, and some notes about interpreting the results.

Fixes #15911 

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by rsalmond** [comment]
@kfaseela @m4ver1k thanks for the feedback but I feel like I'm getting mixed messages. Should I, or should I not, make some sort of change to snips.sh? Note that at present the changes to that file are the result of running `make gen`.

File: content/en/docs/setup/install/multicluster/verify/snips.sh [hunk 1]
```diff
@@ -20,6 +20,16 @@
 #          docs/setup/install/multicluster/verify/index.md
 ####################################################################################################
 
+snip_verify_multicluster_1() {
```
[the remark this answers is not in the document]
Comment (rsalmond) [reply content/en/docs/setup/install/multicluster/verify/snips.sh:23]: I definitely underestimated how complex the docs tests are. I _think_ I added it to the right place, but I don't see anything obvious in the multicluster test output that suggests this test was executed.

**Comment by m4ver1k** [comment]
> @kfaseela @m4ver1k thanks for the feedback but I feel like I'm getting mixed messages. Should I, or should I not, make some sort of change to snips.sh? Note that at present the changes to that file are the result of running `make gen`.

Apologies I mistook them to be edited and not auto-generated. Please ignore my previous comment.

File: content/en/docs/setup/install/multicluster/common.sh [hunk 2]
```diff
@@ -101,6 +101,9 @@ function cleanup_cluster2
 # between CLUSTER1 and CLUSTER2.
 function verify_load_balancing
 {
+  # Verify istiod is synced
+  snip_verify_multicluster_1
```
Comment (kfaseela) [inline content/en/docs/setup/install/multicluster/common.sh:105]: simply calling verify_multicluster wont be enough.. you will have to compare the output of the command to the expected output which is there in snip_verify_multicluster_1_out  . Take a look at the usages of _verify_like() in other similar tests

**Review by kfaseela — APPROVED** [review]

**Pull request merged**